### DISCUSSION TITLE
[Projects Beta] Allow bulk assigning of issues to Project

### DISCUSSION THREAD

**Discussion by gdavis** [community/community#13381] [Projects and Issues] [answered]
I would like to be able to add multiple issues to a project from a repository. Currently, I have to manually add each issue to the project which makes large lists of issues quite tedious. When you select all issues, the "Projects" drop down disappears, and should remain and show projects that have already been used in the repo.

Here's an example of how it looks now:
![image](https://user-images.githubusercontent.com/110691/159807775-1e03ab86-c0fd-4e39-b1ff-cdb217de077f.png)

**Comment by schalkneethling** [comment]
Looks like it is in beta?

![Screenshot off issue board showing multiple selected issues and the option to all to a project board](https://user-images.githubusercontent.com/10350960/162638673-6bd9e74f-6c39-4817-a9bb-d5f8b1f82e6a.png)

**Reply by schalkneethling** [reply]
Indeed it is:

![Green banner with the words, "8 issues have successfully been added to 2 projects"](https://user-images.githubusercontent.com/10350960/162638858-e91ad9de-be66-487c-86aa-21f52921011f.png)

**Comment by JannesPeters** [comment, marked as answer]
This feature was added in April 2022 - https://github.blog/changelog/2022-04-07-the-new-github-issues-april-7th-update/

**Reply by michael-hawker** [reply]
Selection in the issue view only selects the current page of issues. If I have multiple pages of issues to assign to a project (or label, or milestone, or type), how do I select beyond the visible page and actually select all of them to manipulate?
